src/orbital-helpers/fragment_projector.h

```diff
@@ -56,8 +56,16 @@ class FragmentProjector {
     // std::shared_ptr<BasisSet> minao_basis, 	std::shared_ptr<BasisSet> prime_basis);
 
     // Build projector and return AO basis matrix Pf_AO
-    psi::SharedMatrix build_f_projector(std::shared_ptr<psi::Molecule> molecule,
-                                        std::shared_ptr<psi::BasisSet> basis);
+	psi::SharedMatrix build_f_projector(std::shared_ptr<psi::BasisSet> basis);
+
+    /// Return number of basis functions in the system
+    int get_nbf_A() { return nbf_A_; }
+
+    /// Return number of atoms in the system
+    int get_natom_A() { return natom_A_; }
+
+    /// Return AO overlap S_nn
```

Comment:
```suggestion
    /// Return the AO overlap matrix (S_nn)
```

what does `_nn` stand for?

Reply:
It stands for a "square matrix". I think it is not necessary and I will remove this note.